src/test/data/JsonAddressBookStorageTest/invalidAndValidPersonAddressBook.json

```diff
@@ -3,11 +3,45 @@
     "name": "Valid Person",
     "phone": "9482424",
     "email": "hans@example.com",
+    "schedule" : {
+      "events" : [ {
+        "eventDescription" : {
+          "eventDescription" : "CS2103T Tutorial"
+        },
+        "date" : "2022-12-28",
+        "time" : "10:00:00",
+        "duration" : "PT3H"
+      }, {
+        "eventDescription" : {
+          "eventDescription" : "CS2106 Lab"
+        },
+        "date" : "2022-12-28",
+        "time" : "10:00:00",
```

Comment:
Em, perhaps a timing that does not clash with "CS2103T Tutorial" will be better here